.github/workflows/ci.yml

```diff
@@ -68,14 +70,9 @@ jobs:
           rustup default stable
 
       - name: Install rustup-toolchain-install-master and xargo
-        # to avoid saving rustup proxy in .cargo/bin, save CLIs in .cargo/ci/bin.
         run: |
-          echo "$HOME/.cargo/ci/bin" >> $GITHUB_PATH
-          cargo install --root ~/.cargo/ci rustup-toolchain-install-master
-          cargo install --root ~/.cargo/ci xargo
-
-          # Strip files in `ci/bin` dir to remove useless debug-info.
-          strip ~/.cargo/ci/bin/*
```

Comment:
FWIW, you can still do the stripping if you want

Reply:
Nah, it's fine, as long as you're comfortable with the change.